webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/batch/AsyncDeploymentActor.scala

```diff
@@ -436,6 +452,27 @@ final class AsyncDeploymentActor(
       updateListeners()
     }
 
+    //
+    // Updating state from technique compilation output result
+    //
+    case UpdateCompilationStatus(status)                                                   => {
+      StatusLoggerPure.Techniques.logEffect.trace(
+        s"Compilation status has a new update command with payload : ${status}"
+      )
+      if (compilationStatus != status) {
+        compilationStatus = status
+        status match {
+          case CompilationStatusErrors(techniquesInError) =>
+            StatusLoggerPure.Techniques.logEffect.info(s"Status of technique compilation has ${techniquesInError.size} errors")
```

Comment:
I think it should be error level. Or at least warn. 
Yes, warn. It's not a rudder bug, but something that is not normal (ie not the normal error flow of rudder, technique should always compile).

Reply:
yes that is right :white_check_mark: https://github.com/Normation/rudder/pull/5905/commits/394d64c80db4aa00c84fe5793449093c3b48ab76